recipes/default.rb

```diff
@@ -15,4 +16,29 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-#
+
+package 'yum-cron'
+
+%w(daily hourly).each do |t|
+  if node['yum-cron']["#{t}_conf"]
```

Comment:
Rubocop want `next if node['yum-cron']["#{t}_conf"]`
Foodcritic wants `node['yum-cron']["#{t}_conf"]` to be a boolean value.